var/spack/repos/builtin/packages/vtk-m/package.py

```diff
@@ -55,12 +55,6 @@ class VtkM(CMakePackage, CudaPackage, ROCmPackage):
 
     variant("doubleprecision", default=True, description="enable double precision")
     variant("logging", default=False, when="@1.3:", description="build logging support")
-    variant(
```

Comment:
For older versions of VTK-m we still need this right?

Reply:
I thought it was an alias for `VTKm_USE_DOUBLE_PRECISION` and `VTKm_USE_64BIT_IDS`, and I did not see `ascent` actually requiring `+ascent_types` so I assumed it was unused.

Reply:
VTKm_USE_DOUBLE_PRECISION and VTKm_USE_64BIT_IDS were introduced in vtk-m version 1.0.0, so those are generally supported.

And I made ascent force 32 bit ints and double precision, which is what `VTKm_USE_DEFAULT_TYPES_FOR_ASCENT` aliases to.

Reply:
hmm..I think maybe you are right on the compiled code. It does change the installed CMake package configuration file which is reason to keep this flag since it still changes the installed package even if it isn't related to headers/binaries.

Reply:
Turning on the Ascent types in the config automagically in the correct scenario works for me.